test/compositemark/errorbar.test.ts

```diff
@@ -381,28 +381,26 @@ describe('normalizeErrorBar with raw data input', () => {
       assert.fail(!layer, false, 'layer should be a part of the spec');
     }
   });
+});
 
+describe('normalizeErrorBar for all possible extents and centers with raw data input', () => {
   const centers: ErrorBarCenter[] = ['mean', 'median', undefined];
   const extents: ErrorBarExtent[] = ['stderr', 'stdev', 'ci', 'iqr', undefined];
 
   const warningOutput: boolean[][][] = [
-    [
-      [false, false], [false, false], [false, true], [true, true], [false, false]
-    ],
-    [
-      [true, false], [true, false], [true, true], [false, true], [false, false]
-    ],
-    [
-      [false, false], [false, false], [false, false], [false, false], [false, false]
-    ]
+   //    stderr           stdev           ci            iqr         undefined
+    [[false, false], [false, false], [false, true], [true, true], [false, false]],   // mean
```

Comment:
Whoa -- this test seems very unreadable.

Reply:
Can you instead refactor it to use dictionary / map?

Reply:
Yes, it is now using dictionary.